### DISCUSSION TITLE
Ortho Creation

### DISCUSSION THREAD

**Discussion by kubilus1** [kubilus1/autoortho#6] [General]
Discussion around creation of orthophotos.

**Comment by Rylabics** [comment]
Hi 
Could you please say something about the kind (bing, google, etc.) of ortho you can use and the zoom level? How long does the process increase the starting time of xplane? how is the status with XP12, are there some stuttering effects because of the different philosophy between XP11 and XP12? I only find a demostration video in internet, could you please tell me if there are other videos showing the results?
Thx!

**Comment by ivotaHR** [comment]
https://simheaven.com/
